<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>joinGroupDialog</class>
 <widget class="QDialog" name="joinGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Group Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="groupNameLineEdit">
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="joinPushButton">
        <property name="text">
         <string>Join Group</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createPushButton">
        <property name="text">
         <string>Create Group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>label</zorder>
  <zorder>groupNameLineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>createGroupLineEdit</zorder>
  <zorder>line</zorder>
  <zorder>horizontalSpacer_3</zorder>
  <zorder>groupNameLineEdit</zorder>
  <zorder>cancelPushButton</zorder>
  <zorder>cancelPushButton</zorder>
  <zorder>groupNameLineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
